docs/Developers/Newcomers_Guide.rst

```diff
@@ -3,61 +3,68 @@
+**Please do read the guide before asking us, as most questions we get are
+covered in here.**
+
+Before you can start working there are a few things you need to know about
+coala:
+
+  - Everyone in the coala community is expected to follow our
+    `Code of Conduct <http://coala.io/coc>`_.
+  - Do not work on any issue without assignment. Someone else might be working
+    on it already.
+  - When something is unclear try to find a solution yourself. This is a
+    valuable skill!
+  - Only after you tried to solve a problem yourself ask the
+    `chat <coala.io/chat>`_
 
 You will start as a newcomer, which is kind of a trial. If you complete the
 following tasks, you will become a developer at coala:
 
-  - run coala on a project of yours
-  - merge a ``difficulty/newcomer`` Pull Request
-  - review at least a ``difficulty/newcomer`` Pull Request
-  - merge a ``difficulty/low`` Pull Request
-  - review at least a ``difficulty/low`` or higher Pull Request
-
-When you ran coala on a project, please fill our
-`usability survey <http://coala.io/usability>`_. Once you got your first Pull
-Request merged successfully, fill in our
-`survey form <http://coala.io/newform>`_. With that you can help us making your
-experience better!
-
-Once you have achieved all these, just ask for being promoted on the chat and
-provide links to your reviews and merged Pull Requests. Then, you will be able
-to name yourself a coala developer!
+  - Run coala on a project of yours and fill our
+    `usability survey <http://coala.io/usability>`_.
+  - Merge a ``difficulty/newcomer`` Pull Request and fill in our
+    `survey form <http://coala.io/newform>`_.
+  - Review at least a ``difficulty/newcomer`` Pull Request.
+  - Merge a ``difficulty/low`` Pull Request.
+  - Review at least a ``difficulty/low`` or higher Pull Request.
 
 .. note::
 
-    **Do not only fix a newcomer issue!** Supervising newcomers is really a lot
-    of work. We're all volunteers and we can't keep this up if you don't help
-    us in other areas as well!
+    Just accepting a PR does not count as a review. Even if that PR is
+    in a mergable state. We want you to learn and develop an eye for finding
+    improvements. When in doubt, don't give an ack and keep reviewing.
 
-Of course, the order is not important, although, we recommend you to start
-with a ``newcomer`` issue, end with a ``low`` issue, and review other PRs in
-the meantime!
-
-This is a step-based guide that will help you get your first contribution
-at coala, making you familiar with the work flow!
+Once you have achieved all these, just ask for being promoted on the chat and
+provide links to your reviews and merged Pull Requests.
+Then, you will be able to name yourself a coala developer!
 
-For more information about Pull Requests, keep reading!
+To see more information about the rights and duties of the different coala
+ranks, see the
+`membership cEP <https://github.com/coala/cEPs/blob/master/cEP-0001.md>`_.
```

Comment:
well cEP is what we intend to do and can be merged without the docs being adjusted IMO, so first change the cEP then the documentation.

That's just my opinion though if someone has a better idea dont hold it back